UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs

```diff
@@ -73,6 +85,18 @@ public void AddDrunkTime(float time)
 			}
 		}
 
+		public void BlindnessValue(bool IsBlind)
```

Comment:
add the summary. Also rename the parameter to ``isBlind`` with lower case ``i``.